terraform/gcp/network.tf

```diff
@@ -68,3 +68,50 @@ module "cloud-nat" {
   name                               = "juniper-nat-config"
   source_subnetwork_ip_ranges_to_nat = "ALL_SUBNETWORKS_ALL_IP_RANGES"
 }
+
```

Comment:
can you add a comment here explaining what's going on or a link to docs?  the priority numbers are, um, interesting :)

Reply:
alaric never got back to me, so I'll make this a TODO in the future